README.md

```diff
@@ -73,6 +74,7 @@ The following arguments can be passed to kured via the daemonset pod template:
 
 ```
 Flags:
+      --annotationTTL time                  force clean annotation after this ammount of time (default 0, disabled)
```

Comment:
Maybe it makes sense to rename the flag to `--annotation-ttl` to be consistent with the others?

Reply:
Yeah totally makes sense.